geoportal/tests/config.yaml

```diff
@@ -11,3 +11,7 @@ vars:
       backend: dogpile.cache.memory
     obj:
       backend: dogpile.cache.memory
+  vectortile:
+    srid: 21781
+    extent: [599900, 199950, 600100, 200050]
+    resolutions: [4000, 2000, 1000, 500, 250, 100, 50, 20, 10, 5, 2.5, 1, 0.5, 0.25, 0.1, 0.05]
```

Comment:
This should also be added in the `vars.yaml_tmpl`
vectortile should be added as `CONFIG_VARS`
and the structure should be added in the `CONST_config-schema.yaml`